apis/object/v1alpha2/types.go

```diff
@@ -97,6 +97,14 @@ type ObjectSpec struct {
 	ForProvider       ObjectParameters   `json:"forProvider"`
 	References        []Reference        `json:"references,omitempty"`
 	Readiness         Readiness          `json:"readiness,omitempty"`
+	// Watch enables watching the referenced or managed kubernetes resources.
+	//
+	// THIS IS AN ALPHA FIELD. Do not use it in production. It is not honored
+	// unless the relevant Crossplane feature flag is enabled, and may be
```

Comment:
maybe be explicit saying how the feature gate is called? `"watches" feature gate`